test/forms/UserEmailFormSpec.scala

```diff
@@ -7,23 +7,48 @@ class UserEmailFormSpec extends PlaySpec {
   "UserEmailForm" should {
 
     "bind when all valid data is supplied" in {
-
+      val form = UserEmailForm().bind(
+        Map(
+          "email" -> "bta4life@hmrc.gov.uk"
+        )
+      )
```

Comment:
Need an assertion here to test it's a success (maybe refer to some of the other form specs